python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -6,38 +6,47 @@
 
 from ray.rllib.models.model import Model
 from ray.rllib.models.fcnet import FullyConnectedNetwork
+from ray.rllib.models.two_level_fcnet import TwoLevelFCNetwork
+from ray.rllib.models.fcnet import MODEL_CONFIGS as FCN_CONFIGS
+from ray.rllib.models.two_level_fcnet import MODEL_CONFIGS as TFCN_CONFIGS
 from ray.rllib.models.action_dist import Reshaper
 
+# TODO(cathywu) support more network types
+MODEL_CONFIGS = {"FullyConnectedNetwork": FCN_CONFIGS,
```

Comment:
Also this is my bad, but I would re-name `MODEL_CONFIGS` to something else, because it's used in a different way from in the other model classes.